### ISSUE TITLE
release on f-droid?

### ISSUE THREAD

**Issue by AndeAndreas31415** [FFDA/SourCherry#18]
Is a release planned on f-droid?

**Comment by FFDA** [comment]
Hello.

Yes, I thinking about it for some time now, but I just don't find the time to do it. And I  definitely need to add SourCherry to F-Droid, because at some point the app might be removed from the PlayStore by Google, so it would become the easiest way to get the app/updates.

In the mean time if you really need to use F-Droid to get it, you can try to use https://apt.izzysoft.de/ repo. I don't exactly know how it end up there. I guess some one had to make a request for SourCherry to be added there, however as I understand it the apk files are taken directly from GitHub.

**Comment by FFDA** [comment]
SourCherry won't be released on F-Droid. You can read through merge request discussion [here](https://gitlab.com/fdroid/fdroiddata/-/merge_requests/14349).
TL;DR: they don't like the license of one of the dependencies the app uses.
